challengeTwo/objcts.js

```diff
@@ -0,0 +1,10 @@
+var k = 0;

```

Comment:
What were you trying to achieve here?
If you were trying to attempt the problem of making an object whose keys are numbers from 1 to 15 and values squares then this isn't the correct way. 
Try the approach below:

1. Create an empty object
2. Loop through numbers from 1 to 15
3. For each number add that number as the key to the object and its square as the value
4. Log your object after the loop has completed.

Hints:
1. How do we create an empty object?
One way to do it in js is
```js
var myNewObject = {}
```
2. How do we add key-value pairs to objects?
I will cover a case where we first create an object and add key-value pairs to it from variables since its what you need to tackle this. We can talk about the rest later.
```js
// given an empty object below
var person = {}

// and two variables we want to use as a key-value pair below
var myKey = "age"
var myValue = 13

// we can add the key value pair to the object like
person[myKey] = myValue

console.log(person) // logs {age: 13} to the console
```